tests/e2e/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
       "luxon": "^2.4.0"
+  },
+  "devDependencies": {
+    "xml-js": "^1.6.11"
```

Comment:
So that means @SagarGi's PR has not been merge yet?

Reply:
Not yet.